hushline/model.py

```diff
@@ -33,17 +34,17 @@ class User(Model):
     id: Mapped[int] = mapped_column(primary_key=True)
     primary_username: Mapped[str] = mapped_column(db.String(80), unique=True)
     display_name: Mapped[Optional[str]] = mapped_column(db.String(80))
-    _password_hash: Mapped[str] = mapped_column("password_hash", db.String(512))
+    _password_hash: Mapped[str] = mapped_column("password_hash", db.String(512), nullable=True)
```

Comment:
I believe the proper way to indicate nullable using declarative table mapping is
```suggestion
    _password_hash: Mapped[Optional[str]] = mapped_column("password_hash", db.String(512))
```